src/commands/functions/create.js

```diff
@@ -197,22 +197,32 @@ const pickTemplate = async function () {
 const DEFAULT_PRIORITY = 999
 
 /* get functions dir (and make it if necessary) */
-const ensureFunctionDirExists = function (context, flags, config) {
-  const functionsDir = config.build && config.build.functions
-  if (!functionsDir) {
-    context.log(`${NETLIFYDEVLOG} No functions folder specified in netlify.toml`)
-    process.exit(1)
+const ensureFunctionDirExists = async function (context, flags, config) {
+  let functionsDirHolder = config.build && config.build.functions
+  if (!functionsDirHolder) {
+    context.log(`${NETLIFYDEVLOG} functions folder not specified in netlify.toml or UI settings`)
+    const { functionsDir } = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'functionsDir',
+        message: 'Enter the path where your functions should live: netlify/functions:',
```

Comment:
```suggestion
        message: 'Enter the path where your functions should live:',
```

Per you screenshot it seems we don't need to print `netlify/functions` twice.